src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -204,6 +204,7 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
       const isBridge = swapsStore.getState().inputAsset?.mainnetAddress === swapsStore.getState().outputAsset?.mainnetAddress;
       const isDegenModeEnabled = swapsStore.getState().degenMode;
       const slippage = swapsStore.getState().slippage;
+      const isSwappingToPopularAsset = swapsStore.getState().outputAsset?.sectionId === 'popular';
```

Comment:
is `sectionId` an internal prop in FlashList? I don't see it being set anywhere.

Reply:
nvm github is mental. Checked out the code and see it.